x/auth/ante.go

```diff
@@ -106,7 +110,6 @@ func NewAnteHandler(am AccountKeeper, fck FeeCollectionKeeper) sdk.AnteHandler {
 				return newCtx, res, true
 			}
 
-			// Save the account.
```

Comment:
why removing this comment? isn't it true that this saves the account?

Reply:
indeed it is true...it's so true that it's also so obvious haha. I removed it because it was so obvious...i.e. a very redundant comment.